### DISCUSSION TITLE
用transifex搭个板块专门搬运和翻译国外的白皮书？

### DISCUSSION THREAD

**Discussion by HEFZZJM** [FRC-China/FRC-Wiki#24] [Ideas]
rt，虽然现在说好像有点晚了。CD上有好多零零碎碎的white paper（像这篇[second order kinemic model的white paper ](https://www.chiefdelphi.com/t/whitepaper-swerve-drive-skew-and-second-order-kinematics/416964)）。可以用transifex搭个板块专门负责大批量搬运和翻译国外的papers啥的。

**Comment by HEFZZJM** [comment]
https://6940swerve-docs.readthedocs.io/zh_CN/latest/
之前我们翻了个1684的swerve doc（但是结果WPILib内置了swerve运算，所以基本上就没啥用了），但可以参考一下

**Comment by CLCK0622** [comment]
收到 后续可以考虑
但是我个人认为单纯的翻译不如经过融合和二创然后直接贡献到wiki上
